pkg/kubeflags/data.go

```diff
@@ -30,7 +30,6 @@ var defaultAdmissionControllers = []string{
 	"RuntimeClass",
 	"CertificateApproval",
 	"CertificateSigning",
-	"ClusterTrustBundleAttest",
```

Comment:
Why is this admission controller removed? I still see it as enabled by default in the API server.

Reply:
In some doc on kubernetes.io regarding admission controllers it was missing from the list. But since it's present in CLI reference I guess that list is the correct list. Returned it back: :heavy_check_mark: